package.json

```diff
@@ -34,9 +34,12 @@
     "@manuscripts/library": "1.3.7",
     "debug": "^4.3.4",
     "jszip": "^3.10.1",
+    "lodash": "^4.17.21",
+    "migration-base": "npm:@manuscripts/transform@2.3.9-LEAN-3577-support-merge",
```

Comment:
Can this be a dev dependency?

Reply:
Yes, we need it only for running tests. Moved.